iclr2017/webfield/areachair_webfield.html

```diff
@@ -0,0 +1,215 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <div id='main'>
+      <div></div>
+        <h1>Areachair Console</h1>
+      <div id='tablepanel' class='panel'></div>
+    </div>
+    <script type="text/javascript">
+    $(function() {
+ 
+      var sm = mkStateManager();
+
+      var httpGetP = function(url, queryOrBody) {
+        var df = $.Deferred();
+        httpGet(url, queryOrBody, 
+        function(result) {
+          df.resolve(result);
+        },
+        function(result) {
+          df.reject(result);
+        });
+        return df.promise();
+      };
+
+      var getClass = function(colIndex){
+        if(colIndex==0){
+          return 'paper_summary';
+        } else if(colIndex==1){
+          return 'review_progress';
+        } else if(colIndex==2){
+          return 'status';
+        } else {
+          return null;
+        }
+      }
+
+      var mkReviewProgress = function(reviewerGroup,paperReviews,paperQuestions){
+        var numReviewers = reviewerGroup.members.length;
+        var numPaperReviews = paperReviews ? paperReviews.length : 0;
+        var numPaperQuestions = paperQuestions ? paperQuestions.length : 0;
+
+        if(numReviewers > 0){
+          var ratio = "<b>Progress:</b> "+numPaperReviews+" out of "+numReviewers+" official reviews submitted.";
+          paperNum = reviewerGroup.id.split('paper')[1].split('/')[0];
+          //var reviewers = "<b>Reviewers:</b>\n";
+          var $reviewers = $('<div>',{id:paperNum,html:'<b>Reviewers:</b>\n'});
+
+          _.forEach(reviewerGroup.members, function(reviewer){
+            httpGetP('groups',{id:reviewer}).then(function(reviewerResult){
+              $('#'+paperNum).append(view.prettyId(reviewer).split(' ')[4])
```

Comment:
Isn't there other way to get a paper number? a regex over the id? If we change the prettyId function this will fail.